crates/apub/src/objects/mod.rs

```diff
@@ -75,6 +75,7 @@ pub(crate) mod tests {
   use reqwest_middleware::{ClientBuilder, Middleware, Next};
   use std::sync::{Arc, Mutex};
   use task_local_extensions::Extensions;
+  use tokio::sync::broadcast;
```

Comment:
You don't need to do that for this, but something to consider for pretty much all our singletons, services, etc.

Reply:
I feel like these actors would be more heavyweight and complex, because they can send and receive arbitrary messages. On the other hand, these tokio channels can only send one specific message type in a specific direction.